package.json

```diff
@@ -1,35 +1,44 @@
 {
   "name": "medic-couch2pg",
-  "version": "1.2.1",
+  "version": "1.3",
```

Comment:
2.0 plz, you've removed stuff, changed the way the CLI works etc, it definitely is an entirely new deal.